app/views/spotlight/admin_users/index.html.erb

```diff
@@ -5,20 +5,26 @@
     <table class="table table-striped">
       <thead>
         <tr>
-          <th colspan="2"><%= Spotlight::Engine.user_class.human_attribute_name(:email)  %></th>
+          <th scope="col"><%= Spotlight::Engine.user_class.human_attribute_name(:email) %></th>
+          <th scope="col"><%= t('.site_admin') %></th>
         </tr>
       </thead>
-      <tbody class="table">
+      <tbody>
         <% @site.roles.map(&:user).each do |user| %>
           <tr>
-            <td class="<%= 'invite-pending' if user.invite_pending? %>">
+            <td class="<%= user_badge_classes(user) %>">
               <%= user.email %>
+              <span class="badge bg-secondary site-admin-label"><%= t('.site_admin').downcase %></span>
               <span class='badge bg-warning pending-label'><%= t('.pending') %></span>
             </td>
             <td>
-              <%= link_to(t('.destroy'), admin_user_path(user),
-                    data: { method: :delete, turbo_method: :delete },
-                    class: 'btn btn-sm btn-danger float-end') unless user == current_user %>
+              <%= link_to(t('.destroy'), spotlight.admin_user_path(user),
+                    data: {
+                      method: :delete,
+                      turbo_method: :delete,
+                      turbo_confirm: t('.destroy_confirm', email: user.email)
+                    },
+                    class: 'btn btn-sm btn-danger') unless user == current_user %>
```

Comment:
It looks like we're duplicating this part of the UI here and below. Would it make sense to extract this email, badge, and remove role link to a view component? That would also make it so we're not adding new helper method.

Reply:
Looks great. Thank you for making these changes.